### PR TITLE
Create journal-of-egyptian-archaeology.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/59515/style-request-journal-of-egyptian-archaeology
Adapted for journals, books, chapters, web
Citation (footnote) only style.

### CODE REVIEW

File: the-journal-of-egyptian-archaeology.csl [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    <link href="http://www.zotero.org/styles/journal-of-the-history-of-collections" rel="template"/>
+    <link href="http://www.ees.ac.uk/publications/journal-egyptian-archaeology.html" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <issn>0075-4234</issn>
+    <eissn>0307-5133</eissn>
```
Comment (rmzelle) [inline the-journal-of-egyptian-archaeology.csl]: @POBrien333, where did you find the "0075-4234" ISSN? It's associated with the journal somehow, but it's marked as wrong/invalid by several catalogues:

https://www.ncbi.nlm.nih.gov/nlmcatalog/?term=0075-4234&report=xml&format=text

    <ISSN ValidYN="Y" IssnType="Print">0307-5133</ISSN>
    <ISSN ValidYN="N" IssnType="Undetermined">0075-4234</ISSN>
    <ISSNLinking ValidYN="Y">0307-5133</ISSNLinking>
>

http://library.wur.nl/WebQuery/clc/1847031
> ISSN: 0307-5133
> Wrong ISSN: 0075-4234

http://catalogue.nla.gov.au/Record/2269193
> ISSN: 0075-4234
> Invalid ISSN: 0307-5133

Comment (rmzelle) [inline the-journal-of-egyptian-archaeology.csl]: (Unless you have answers about this genuine mystery, 🤔, I would just use 0307-5133 as the print ISSN, per https://www.ncbi.nlm.nih.gov/nlmcatalog/?term=0075-4234, and leave out the other one)

**Comment by rmzelle** [comment]
Thanks!

**Comment by POBrien333** [comment]
Thanks. Couldn't fix that on the phone properly. It wouldn't let me delete that line.

**Pull request merged**